controlplane/kubeadm/api/v1alpha3/kubeadm_control_plane_types.go

```diff
@@ -25,10 +25,11 @@ import (
 )
 
 const (
-	KubeadmControlPlaneFinalizer    = "kubeadm.controlplane.cluster.x-k8s.io"
-	KubeadmControlPlaneHashLabelKey = "kubeadm.controlplane.cluster.x-k8s.io/hash"
-	SelectedForUpgradeAnnotation    = "kubeadm.controlplane.cluster.x-k8s.io/selected-for-upgrade"
-	DeleteForScaleDownAnnotation    = "kubeadm.controlplane.cluster.x-k8s.io/delete-for-scale-down"
+	KubeadmControlPlaneFinalizer         = "kubeadm.controlplane.cluster.x-k8s.io"
+	KubeadmControlPlaneHashLabelKey      = "kubeadm.controlplane.cluster.x-k8s.io/hash"
+	SelectedForUpgradeAnnotation         = "kubeadm.controlplane.cluster.x-k8s.io/selected-for-upgrade"
+	DeleteForScaleDownAnnotation         = "kubeadm.controlplane.cluster.x-k8s.io/delete-for-scale-down"
+	ScaleDownETCDMemberRemovedAnnotation = "kubeadm.controlplane.cluster.x-k8s.io/scale-down-etcd-member-removed"
```

Comment:
We've been using `Etcd`, I think we should stick with that convention unless there's a good reason to change everything else to use `ETCD`

Reply:
I've been trying to stick with the convention that acronyms or single case words stay single cased, but I know other uses have slipped in especially in the case of auto-generated code, where the AWS sdk is probably the worst offender.

Reply:
Seems k/k uses `Etcd` as well, so I'd stick with that 👍